deploy/kustomize/kubernetes-1.14/driverinfo-beta.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: pmem-csi.intel.com
+spec:
+  attachRequired: true
```

Comment:
I think we can drop this. If I am not wrong, Our driver no more needs attachment.

Reply:
Can we do that in a separate PR?

Reply:
If we don't attachments, then we shouldn't be running external-attacher either. This is a separate change than adding support for Kubernetes 1.14.

Reply:
yes, We can do it as separate PR.

Reply:
https://github.com/intel/pmem-CSI/issues/264